### PR TITLE
Implements Vuejs docs

### PR DESCRIPTION
**Pull request by niallobrien**
Initial implementation of the Vue.js docs to coincide with the Feathers & Vue.js chat example.

### CODE REVIEW

File: frameworks/vuejs.md [hunk 1]
```diff
@@ -1,3 +1,328 @@
...
+  }
+})
+```
+
+## Further study
+The Vue documentation is a fantastic source of reference. As your app grows, you may become interested in introducing Vuex, a state management library for Vue, and its based heavily on Redux.
+
+Below is an introduction to using Vuex with Vue and Feathers on the backend.
+
+[![Realtime Vue.js and Feathers.js Example](http://i.imgur.com/MhYLgxb.png)](https://www.youtube.com/watch?v=zbhYcxr5ldk "Realtime Vue.js and Feathers.js Example ")
```
Comment (ekryski) [inline frameworks/vuejs.md]: You can actually embed the youtube video right in there.

File: frameworks/vuejs.md [hunk 2]
```diff
@@ -1,3 +1,328 @@
...
+
+> **ProTip:** This setup is not recommended for a production environment. Instead, it is recommended that you use Webpack to compile your client-app or [Vue-CLI](https://github.com/vuejs/vue-cli) which can scaffold an app for you, complete with build tools etc. You then just need to use the Feathers client modules [in the client chapter](../clients/feathers.md).
+
+## Application Bootstrap
+
+All of the code examples that follow should be appended to the `public/vue-chat/app.js` file, which the HTML file already loads. The first step is to add a placeholder user in the event that a user image or other user information doesn't exist:
+
+```javascript
+// A placeholder image if the user does not have one
+const PLACEHOLDER = '/placeholder.png';
```
Comment (ekryski) [inline frameworks/vuejs.md]: We just updated the placeholder url so that it pulls from a remote endpoint.

**Comment by ekryski** [comment]
:shipit: Man this is awesome! Nice work @niallobrien!! 🎉 

Don't worry about my comments I'll just merge and fix those two things right away.

**Comment by niallobrien** [comment]
Thank you. :)

**Pull request merged**